dt-assets/parts/nav-topbar.php

```diff
@@ -19,6 +19,13 @@
  */
 $dt_nav_tabs = dt_default_menu_array();
 
+/**
+ * Ensure people groups post type is displayed accordingly based on option setting.
+ */
+
+if ( isset( $dt_nav_tabs['main']['peoplegroups'] ) && ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab ) ) {
+    unset( $dt_nav_tabs['main']['peoplegroups'] );
+}
```

Comment:
@kodinkat fron the people-groups-base, hook into the dt_nav filter and set the hidden to true
This way other configs can also have a say.